src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -130,5 +140,16 @@ public async Task<IActionResult> GetDatamodel(string org, string repository, str
                 return NotFound();
             }
         }
+
+        private bool ValidateFilePath(string filepath)
+        {
+            string[] pathParts = filepath.Split('/');
+            if (pathParts.Length != 3 || pathParts[0] != "App" || pathParts[1] != "models")
```

Comment:
Is it possible to get rid of the App/models dependency in frontend?

Reply:
I've made changes so that the logic now use model name instead of file path.